### PR TITLE
Added UserDelegate Get method

### PR DESCRIPTION
**Pull request by leobcgov**
# Fixes or Implements [AB#9241](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/9241)

* [ ] Enhancement
* [ ] Bug
* [ ] Documentation

## Description
- Added Get method to the UserDelegate delegate.

## Testing

* [ ] Unit Tests Updated
* [ ] Functional Tests Updated
* [X] Not Required

### Steps to Reproduce

If this is a bug, please provide details on how to reproduce the code.

### UI Changes

NO

### Browsers Tested

* [ ] Chrome
* [ ] Safari
* [ ] Edge
* [ ] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments

Any additional notes or comments that may be relevant.  Include any specialized deployment steps here.  

</br>
</br>
</br>
___
  
## Reviewer Guidelines

A reviewer should examine the PR if the Status above indicates the PR is **Ready**.
Once ready, the reviewer should read the information below as well as supporting information like the Azure ticket to ensure they have a good understanding of the request.

The reviewer should then:

1. Is the PR template complete and accurate?

    Confirm by reading all sections below.

1. Confirm the merge branch

    A normal feature/bug branch should merge into the Dev branch.  Exceptions for hotfix and sprint merges do occur and the target would then be Master.

1. Do the code commits have an appropriate comment and include linkage to the azure ticket by including a reference to "AB#nnnn"?

1. Review the Code Changes

    1. Do the changes match the description?

        The code commits should reflect the documented description and nothing more.  If additional changes are included the reviewer should call these out and request that the developer document the changes in Azure and in the description or remove the changed code.

    1. useful tips

        * Any obvious errors or omissions?
        * Does the code match the style of other similar work?
        * Is the code easy to understand?
        * Is the code well documented?
        * DB Migrations are appropriate?

If the Answer to any of the questions above is No, then a request for change should be the result of the review.

### CODE REVIEW

File: Apps/Database/src/Delegates/DBUserDelegateDelegate.cs [hunk 1]
```diff
@@ -73,5 +73,16 @@ public DBResult<UserDelegate> Insert(UserDelegate userDelegate, bool commit = tr
             this.logger.LogDebug($"Finished inserting user delegate to DB... {JsonSerializer.Serialize(result)}");
             return result;
         }
+
+        /// <inheritdoc />
+        public DBResult<IEnumerable<UserDelegate>> Get(string delegateId)
+        {
+            this.logger.LogTrace($"Getting dependents from DB...");
+            return DBDelegateHelper.GetPagedDBResult(
```
Comment (tiagograf-AOT) [inline Apps/Database/src/Delegates/DBUserDelegateDelegate.cs:81]: Add Log debug with the result set

File: Apps/Database/src/Delegates/DBUserDelegateDelegate.cs [hunk 2]
```diff
@@ -73,5 +73,16 @@ public DBResult<UserDelegate> Insert(UserDelegate userDelegate, bool commit = tr
...
+                this.dbContext.UserDelegate
+                    .OrderBy(dependent => dependent.DelegateId == delegateId),
+                1,
+                100);
```
Comment (tiagograf-AOT) [inline Apps/Database/src/Delegates/DBUserDelegateDelegate.cs:85]: introduce constants

File: Apps/Database/src/Delegates/DBUserDelegateDelegate.cs (hunk 1 above)
Comment (FuriousLlama) [reply Apps/Database/src/Delegates/DBUserDelegateDelegate.cs:81]: +1

File: Apps/Database/src/Delegates/DBUserDelegateDelegate.cs (hunk 2 above)
Comment (FuriousLlama) [reply Apps/Database/src/Delegates/DBUserDelegateDelegate.cs:85]: +1

Comment (leobcgov) [reply Apps/Database/src/Delegates/DBUserDelegateDelegate.cs]: updated

File: Apps/Database/src/Delegates/DBUserDelegateDelegate.cs (hunk 1 above)
Comment (leobcgov) [reply Apps/Database/src/Delegates/DBUserDelegateDelegate.cs]: updated

File: Apps/Database/src/Delegates/DBUserDelegateDelegate.cs [hunk 3]
```diff
@@ -73,5 +73,18 @@ public DBResult<UserDelegate> Insert(UserDelegate userDelegate, bool commit = tr
...
+        /// <inheritdoc />
+        public DBResult<IEnumerable<UserDelegate>> Get(string delegateId, int page, int pageSize)
+        {
+            this.logger.LogTrace($"Getting user delegates from DB...");
+            var result = DBDelegateHelper.GetPagedDBResult(
+                this.dbContext.UserDelegate
+                    .OrderBy(dependent => dependent.DelegateId == delegateId),
+                page,
+                pageSize);
+            this.logger.LogDebug($"Finished getting user delegates to DB... {JsonSerializer.Serialize(result)}");
```
Comment (FuriousLlama) [inline Apps/Database/src/Delegates/DBUserDelegateDelegate.cs:86]: logTrace on the data

**Review by FuriousLlama — APPROVED** [review]

**Review by tiagograf-AOT — APPROVED** [review]

**Pull request merged**